### DISCUSSION TITLE
Operation manuals for a station

### DISCUSSION THREAD

**Discussion by freddepettersson** [tellurianinteractive/Tellurian.Trains.ModulesRegistryApp#27] [Ideas]
I was a little curios regarding idea and future use of the operation instructions  for a station.
In my head I saw the use of adding the instructions when a station are used in a traffic game. But there will be a lot of different PDF styles, language etc.

If the idea in the future are to add the operation manual to a station/ loading depot when the paper work are printed for a traffic game, my thoughts was that it might be a good way to add data in a editor (like here on GitHub) in the module registry page directly . 
Then the text might be used in different reports in the future, the report can adjust layout etc .

**Suggestion**
Add operation instructions to the station like a Editor or free text boxes with different topics
Two alternative, module own can use both but mandatory the shortversion
**_Shortversion_** – can be used in reports or when sending out the game plan before the meeting
**_Long version_** – can be activated when reports for the traffic game are printed. For ex. When printing time schedule, the operator manual can be printed 

If a free text input are selected some info can be good ( more topics /terms) might be needed.
-	How to operate switches
-	How to operate signal
-	Any special functions

/Fredrik

**Comment by fjallemark** [comment]
Sounds like a good idea. I suggest adding a form where you can write [markdown](https://www.markdownguide.org/getting-started/), which then can be printed with the station or shunting instructions. Maybe one field for shunting instructions and one for traffic controller instructions, which is the model I use in my planning today. The information should be general and not tied to any specific operation game sessions. I can also see an need to add similar descriptive content that is specific to one layout, but this has to be made in the planning system.

**Reply by freddepettersson** [reply]
I check Markdown , and yes it migth be useful, BUT the system needs ormust be "Esay to use" - monkey prof , otherwise it just will be a function with a lot of questions .

But main function as you wrote:
- Shunting instruction 
- Traffic control , ( signals/ train routes etc)
- Instructions for traffic planer ( this text are shown when a planner make a layout ( adding module in a layout)  with info regardning "signal modules ", " this track can/must be used in a way"  

The instructions in or for the trafficgame - this must be added by the traffic planer and the input must be in the "planning mode"
